src/components/Breadcrumb/Breadcrumb-story.js

```diff
@@ -19,8 +19,10 @@ storiesOf('Breadcrumb', module).addWithInfo(
     `,
   () => (
     <Breadcrumb {...additionalProps}>
-      <BreadcrumbItem href="#">Breadcrumb 1</BreadcrumbItem>
-      <BreadcrumbItem href="w#">Breadcrumb 2</BreadcrumbItem>
+      <BreadcrumbItem>
+        <a href="/#">Breadcrumb 1</a>
```

Comment:
Added this to show that you can now add a child to the BreadcrumbItem, and if you do then it won't generate an extra `a` tag. Tested it with `react-router` Link as well, and it worked without any warnings.